platform/kobo/koreader.sh

```diff
@@ -266,6 +266,7 @@ CRASH_PREV_TS=0
 # List of supported special return codes
 KO_RC_RESTART=85
 KO_RC_USBMS=86
+KO_RC_HALT=88
```

Comment:
Wikipedia states that [87 (eighty-seven)](https://en.wikipedia.org/wiki/87_(number)) is the natural number following 86 and preceding 88, and I'm inclined to trust it!
So, for what coming up new feature have you reserved yourself 87 for ? :)

Reply:
Cervantes uses 87 to restart to the stock App ;).